examples/fastapi-crud/src/exceptions/database_exceptions.py

```
@@ -4,31 +4,31 @@
-        raise HTTPException(
-            status_code=status.HTTP_409_CONFLICT,
-            detail=detail
-        ) from integrity_error
+   @staticmethod
+   def throw_db_integrity_error(integrity_error: IntegrityError):
+     """Throws SqlAlchemy integrity error detail"""
+     detail = ""
+     if integrity_error.orig.diag.message_detail:
+         detail = integrity_error.orig.diag.message_detail
+     else:
+         detail = str(integrity_error)
```

Ruff (0.12.2)

Use binary operator `detail = integrity_error.orig.diag.message_detail or str(integrity_error)` instead of `if`-`else`-block

Replace `if`-`else`-block with `detail = integrity_error.orig.diag.message_detail or str(integrity_error)`

(SIM108)